sdcm/cluster.py

```diff
@@ -1828,6 +1830,10 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
         self.remoter.run('sudo mv /tmp/scylla.yaml {}'.format(yaml_file))
 
         if append_scylla_args:
+            scylla_help = self.remoter.run("scylla --help", ignore_status=True).stdout
```

Comment:
do we always have "scylla --help" at this stage?

Reply:
As I can see, we expect that scylla installed before the `config_setup()` call because we modify existing files which is a part of scylla package.